src/components/Panel/Panel.Props.ts

```diff
@@ -46,6 +46,11 @@ export interface IPanelProps extends React.Props<Panel> {
    * Aria label on close button
    */
   closeButtonAriaLabel?: string;
+
+  /**
+   * Hide the header text to be only read by screen readers @default false
+   */
+  hideHeaderText?: boolean;
```

Comment:
instead of this, what if we head "screenReaderText" that shows in a separate p, conditionally if provided. Then you can render it separately.